.github/workflows/tests.yml

```diff
@@ -272,7 +219,12 @@ jobs:
           # (sometimes, the install appears to be successful but shared
           # libraries cannot be loaded at runtime, so an actual import is a
           # better check).
-          python -m pip install --upgrade pycairo 'cairocffi>=0.8' 'PyGObject${{ matrix.pygobject-ver }}' &&
```

Comment:
The `pygobject` documentation states that [`cairocffi` is not currently supported](https://pygobject.gnome.org/guide/cairo_integration.html#cairo-integration), so I think we should remove it from the installed dependencies.  Ref: https://gitlab.gnome.org/GNOME/pygobject/-/blob/1c675e47a160662e5247a387dcd35e0d36b3c4d2/docs/guide/cairo_integration.rst

Reply:
(I think the `cairocffi` dependency may be a holdover from the [`pgi`](https://github.com/pygobject/pgi) GTK/PyGObject bindings that were used historically)

Reply:
> (I think the `cairocffi` dependency may be a holdover from the [`pgi`](https://github.com/pygobject/pgi) GTK/PyGObject bindings that were used historically)

Filed as a separate issue, #29822.